### PR TITLE
Adding a bit of info for setting up the buttons.

### PR DESCRIPTION
**Pull request by jcutrell**
This could be a little confusing for anyone who hasn't read about hacking these buttons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ Just add `dashed` to your Gemfile and then run `bundle install`.
 Dashed has to be run with root permissions in order to listen to your network
 devices to pick up your dash button.
 
+## Preparing the buttons
+
+In order to use your [dash buttons](http://www.amazon.com/b/?node=10667898011&sort=date-desc-rank&lo=digital-text) with this library, you will need to set them up using Amazon's setup process. While you can automatically order products using the buttons, if you'd like to prevent the automatic ordering of a product, simply stop the  registration process before choosing a product.
```
Comment (kenyonj) [inline README.md]: Thoughts on pointing people to this instead:
`http://www.amazon.com/oc/dash-button` ?

**Comment by kenyonj** [comment]
Great idea, @jcutrell... Thoughts @BlakeWilliams ?

**Comment by BlakeWilliams** [comment]
This is a good addition. Is there a blog post by someone we could point them to instead so they can have a more in depth guide on setting up the button? I don't think we want this Github repo to become the place for issues regarding setting up the button.

**Comment by kenyonj** [comment]
I think we might be able to point someone to Ted Benson's original medium post that sparked all of these different attempts.
https://medium.com/@edwardbenson/how-i-hacked-amazon-s-5-wifi-button-to-track-baby-data-794214b0bdd8

thoughts @jcutrell ?

**Comment by jcutrell** [comment]
Sounds good to me! That's the one I was referring to myself, so that's probably a good starting point for folks wanting to learn.

**Comment by kenyonj** [comment]
Great. Go ahead and update the PR and I can merge it in.

**Comment by kenyonj** [comment]
Thanks for updating @jcutrell, can you squash those commits into one and just add the info in the body of that one commit? I want to merge it in this morning if possible.
thanks

**Comment by jcutrell** [comment]
That should do the trick, yes?

**Pull request merged**